### DISCUSSION TITLE
Introduction !

### DISCUSSION THREAD

**Discussion by riskyhomo** [MDAnalysis/mdanalysis#4465] [GSoC Discussions]
Hello there!
Iam Saivinayak Akkaldevi currently a 2nd year student at National Insitute of Technology Warangal. Iam have a deep expertise in implementation of research papers and building neural networks . I have been working in this domain for last 6 months and I look very much to contribute to it. I really want to make algorithms to make chemsitry easy and understandable to everyone . I look forward to be a part of GSoC'24 @ MDAnalysis. Iam familiar with Density Functional Theory , build several notebooks on it and deep understanding of computer vision and 3d modeling of molecules . So iam looking forward to contribute to it 

thanking you in advance

**Comment by jennaswa** [comment]
Hi Saivinayak, and welcome to MDAnalysis! We are finalizing our application process for this year's program and will be sharing a post on [our blog](https://www.mdanalysis.org/blog/) with all of the details very soon - stay tuned!
